examples/openspending/components/_shared/Footer.tsx

```diff
@@ -0,0 +1,13 @@
+import Image from 'next/image'
+export default function Footer() {
+  return (
+    <footer>
+      <div className="mx-auto max-w-7xl px-4 sm:px-6 lg:px-8 flex flex-col items-center justify-between md:flex-row">
```

Comment:
I think we should have some text to accompany the Datopian logo

E.g. Maintained by

Reply:
Done